### PR TITLE
Process for Product Training and Q&A Session

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: source/process/training.rst [hunk 1]
```diff
@@ -247,6 +247,91 @@ Frequently Asked Questions
 
 Not everyone is comfortable with speaking in front of 60+ people. Not everyone speaks clear English. We don’t want the first company-wide meeting to be anxious for new team members, hence by default managers intro them. New hires can opt-in to introducing themselves. 
 
+Product Training and Q&A Session
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These are our product training sessions for the Sales and Customer Success teams, along with QA, focused on frequently asked product areas. They include a functional and technical training session targeted as specific audiences.
```
Comment (amyblais) [inline source/process/training.rst]:
```suggestion
These are our product training sessions for the Sales and Customer Success teams, along with QA, focused on frequently asked product areas. They include a functional and technical training session targeted for specific audiences.
```

File: source/process/training.rst [hunk 2]
```diff
@@ -247,6 +247,91 @@ Frequently Asked Questions
...
+
+Time:
+
+- Every 3 weeks on Wednesdays from 9:00am to 10:00am Palo Alto time.
+
+Procedure:
+
+**Before meeting:**
+
+1. (Chair) Schedules a kick-off call with co-chairs, scheduled for two weeks prior to the training session. Purpose is to review expectations and plans for the session, as well as answer any questions.
```
Comment (amyblais) [inline source/process/training.rst]:
```suggestion
1. (Chair) Schedules a kick-off call with co-chairs two weeks prior to the training session. Purpose is to review expectations and plans for the session, as well as answer any questions.
```

File: source/process/training.rst [hunk 3]
```diff
@@ -247,6 +247,91 @@ Frequently Asked Questions
...
+
+3. (Co-Chairs) Two days prior to the training session, practice a dry run for length and structure of the session. This is recommended but optional.
+
+4. (Chair) Queues an item for that week's Customer Obsession meeting to remind everyone of the product training session taking place. Include a mention of the topic, along with who is leading the session.
+
+5. (Chair & Co-Chairs) Signs into their Zoom account via OneLogin to access recording and screenshare capabilities during the meeting.
+
+**During meeting:**
+
+1. (Chair) At 8:58am Palo Alto time on the day meeting is held, posts a reminder of the session along with a Zoom link to join the call. `See example <https://community.mattermost.com/core/pl/ward3bdkii895kw9g4sigbikwo>`_.
```
Comment (amyblais) [inline source/process/training.rst]:
```suggestion
1. (Chair) At 8:58am Palo Alto time on the day the meeting is held, posts a reminder of the session along with a Zoom link to join the call. `See example <https://community.mattermost.com/core/pl/ward3bdkii895kw9g4sigbikwo>`_.
```

File: source/process/training.rst [hunk 4]
```diff
@@ -247,6 +247,91 @@ Frequently Asked Questions
...
+
+2. (Team) Joins the **Zoom** link in the header of the `Product Training and Q&A channel <https://community-release.mattermost.com/core/channels/product-training-and-qa>`_.
```
Comment (amyblais) [inline source/process/training.rst]:
```suggestion
2. (Team) Joins the **Zoom** link in the header of the `Product Training and Q&A channel <https://community.mattermost.com/core/channels/product-training-and-qa>`_.
```

File: source/process/training.rst [hunk 5]
```diff
@@ -247,6 +247,91 @@ Frequently Asked Questions
...
+  - What does it do, why do people use it, who uses it (customer segment) 
+  - Customer use case examples (success stories) 
+  - Known customer issues and workarounds
+  - Roadmap (status and committed timelines, if any) 
+  - Q&A 
+
+- **Technical Training**
+
+  - Audience: Primarily support, SAs and QA
+  - Technical understanding of how the feature works and how it was built (to help with better troubleshoot)
```
Comment (amyblais) [inline source/process/training.rst]:
```suggestion
  - Technical understanding of how the feature works and how it was built (to help with better troubleshooting)
```

**Comment by jasonblais** [comment]
@amyblais I made updates per your feedback + a few additional changes. You can review those additional changes more easily via this commit https://github.com/mattermost/docs/pull/2894/commits/dc842e4dd16a4a0700796a7a10a6401cdd619e0b

This is ready for final editor review, then merge.

**Pull request merged**